README.md

```diff
@@ -1,20 +1,41 @@
+* `libhero-target`: HERO hardware specific APIs accessible throught OpenMP
+* `pulp-sdk`: `bigPULP`accelerator firmware for HERO (more info [here](https://github.com/pulp-platform/pulp-sdk))
+
+## References
+1. [Kurth, Andreas, et al. "HERO: Heterogeneous Embedded Research Platform for Exploring RISC-V Manycore Accelerators on FPGA." _arXiv preprint arXiv:1712.06497 (2017)_](https://arxiv.org/abs/1712.06497)
+2. [Rossi, Davide, et al. "PULP: A parallel ultra low power platform for next generation IoT applications." _Hot Chips 27 Symposium (HCS), 2015 IEEE._ IEEE, 2015](https://ieeexplore.ieee.org/document/7477325/)
+3. [Vogel, Pirmin, Andrea Marongiu, and Luca Benini. "Lightweight virtual memory support for zero-copy sharing of pointer-rich data structures in heterogeneous embedded SoCs." _IEEE Transactions on Parallel and Distributed Systems 28.7 (2017): 1947-1959._ IEEE, (2017)](https://ieeexplore.ieee.org/document/7797491/)
+4. [Capotondi, Alessandro, and Andrea Marongiu. "Enabling zero-copy OpenMP offloading on the PULP many-core accelerator." _Proceedings of the 20th International Workshop on Software and Compilers for Embedded Systems._ ACM, 2017.](https://dl.acm.org/citation.cfm?id=3079071)
+
+# Getting Started
+## Prerequisites
+### Hardware
+HERO SDK — at the moment — supports ufficially only the [Xilinx Zynq ZC706](https://www.xilinx.com/products/boards-and-kits/ek-z7-zc706-g.html) as target hardware platform.
+
+### Software
+#### Ubuntu 16.04
 Starting from a fresh Ubuntu 16.04 distribution, here are the commands to be executed to get all required dependencies:
 ```
-sudo apt install build-essential bison flex git python3-pip gawk texinfo libgmp-dev libmpfr-dev libmpc-dev swig3.0 libjpeg-dev lsb-core doxygen python-sphinx sox graphicsmagick-libmagick-dev-compat libsdl2-dev libswitch-perl libftdi1-dev u-boot-tools fakeroot
+sudo apt install build-essential bison flex git python3-pip gawk texinfo libgmp-dev libmpfr-dev libmpc-dev swig3.0 libjpeg-dev lsb-core doxygen python-sphinx sox graphicsmagick-libmagick-dev-compat libsdl2-dev libswitch-perl libftdi1-dev cmake u-boot-tools fakeroot
+sudo pip3 install artifactory twisted prettytable sqlalchemy pyelftools openpyxl xlsxwriter pyyaml numpy
+```
+#### CentOS 7 (Experimental)
+Starting from a fresh CentOS 7 distribution, here are the commands to be executed to get all required dependencies:
+```
+sudo yum install git python34-pip python34-devel gawk texinfo gmp-devel mpfr-devel libmpc-devel swig libjpeg-turbo-devel redhat-lsb-core doxygen python-sphinx sox GraphicsMagick-devel ImageMagick-devel SDL2-devel perl-Switch libftdi-devel cmake uboot-utils fakeroot
 sudo pip3 install artifactory twisted prettytable sqlalchemy pyelftools openpyxl xlsxwriter pyyaml numpy
 ```
-### Checkout the HERO SDK sources
+
+## Get the HERO SDK sources
 The HERO SDK uses GIT submodule. To checkout properly the sources you have to execute the following command:
```

Comment:
The HERO SDK uses GIT submodule**s**.

> To checkout properly ...

Thus, you have to clone it recursively: